### DISCUSSION TITLE
Mentors List

### DISCUSSION THREAD

**Discussion by Durgesh4993** [Girlscript-WOB/Contributor#11] [Announcements]
|Project Name|Project Link (GitHub) |Project Admin/ Organizataion Name|Mentor Assigned|Mentor's LinkedIn|Second Mentor Assigned|Second Mentor's LinkedIn|
| - | - | - | - | - | - | - |
|Winter of Blockchain Web3 Project|https://github.com/Girlscript-WOB/WoB-Website|Winter Of Blockchain|SAKSHAM SHANDILYA|https://www.linkedin.com/in/shandilya-saksham-full-stack-developer|Somya Sharma |https://www.linkedin.com/in/somya-sharma-b5ba01229|
|Course Guard|https://github.com/Sanskriti1102/CourseGuard|Sanskriti Kadam|Suchitra Sahoo|https://www.linkedin.com/in/suchitra-sahoo-a60a05257/|Harmeet Singh|https://www.linkedin.com/in/harmeetsingh113/|
|web3-Dream11|https://github.com/yogesh0509/web3-DREAM11|Yogesh Agrawal|Vinith n|https://www.linkedin.com/in/vinith-n-775a37259/|Anuj Mishra|https://www.linkedin.com/in/anujmishra05|
|Fundraising app smart contract|https://github.com/Parshav14/Web3\_Blockchain|Parshav Singla|Bama Charan Chhandogi|https://www.linkedin.com/in/bamacharanchhandogi/|Dhanush H M|https://www.linkedin.com/in/dhanush-h-m-8114b1251/|
|DesiNFT|https://github.com/ayush-that/HackOn-Blocks-2024|Ayush Kumar Singh|Afreen A Habeeb|https://www.linkedin.com/in/afreen-a-habeeb/|Kashish Dange|https://www.linkedin.com/in/kashish-dange-35179b27b/|
|Weather Based Application|https://github.com/Manyap772/weather|Manya Pandey|Adnan Mohsin |https://www.linkedin.com/in/muhd-adnan-mohsin-7b01b8252|Shraman Jain|https://www.linkedin.com/in/shraman-jain/|
|AI Autonomous Billing System|https://github.com/AdnanMohsin21/Enigma-\_AMUHACKS3.0|Adnan Mohsin|Nooras Fatima Ansari |https://noorasfatima.netlify.app/|Nitin Chakrawarti |https://in.linkedin.com/in/nitinchakrawarti|
|groovin|https://github.com/GautamAnjali569/grovvin|Anjali Gautam|Shrinjay Shresth |https://www.linkedin.com/in/shrinjay-shresth-036010215/|Akshat Sharma|https://www.linkedin.com/in/akshat-sharma-638b16219/|
|Adopt pet|https://github.com/Swarnendu0123/adopt-pet|Swarnendu|Avdhesh Varshney|https://www.linkedin.com/in/avdhesh-varshney/|Mansi Ruhil |https://www.linkedin.com/in/mansi-ruhil-7a00a0228|
|Decentralised blockchain|https://github.com/ashwincs12/Decentralized-Kudumbashree|Sherhin pp|Shashank Shekhar Singh|https://www.linkedin.com/in/shashank-shekhar-singh-121128254/|Meet Jain|https://linkedin.com/in/meet-jain-413015265/|
|Maargdarshak|https://github.com/seohyeonlee2020/connecting-communities|Ajay Deshmukh|Anuradha Adhikari|https://in.linkedin.com/in/anuradha-adhikari|Navneet Kumar|https://www.linkedin.com/in/mrnavneet/|
|CRYPTOANALYSIS|**https://github.com/Lakshya1726/CRYPTOANALYSIS**|Lakshya|YASH YADAV|https://www.linkedin.com/in/yash-yadav23/|Unnati gupta|https://www.linkedin.com/in/unnati-gupta-b73a8a27b|
|LegaLInk|https://github.com/Meghnagit20/LegaLINc|Meghna Malasi|Mustafa Hassan|https://www.linkedin.com/in/mustafahassan2001/|Hari Varshney|https://www.linkedin.com/in/hari2005/|
|CourseGuard|https://github.com/WEBXELA/CourseGuard|Sanskriti Kadam|Krishi Shah|https://www.linkedin.com/in/krishi-shah-1n2ov/|LC Stroud|https://www.linkedin.com/in/lcstroud/|
|GrubHub|https://github.com/Paramhans-Singh/GrubHub|Paramhans Singh|Amarjeet |https://www.linkedin.com/in/jeetamar|Sahil Singh|www.linkedin.com/in/sahil-singh-653561239|
|Project-Guidance|https://github.com/Kushal997-das/Project-Guidance|Kushal Das|Abhishek Kumar Yadav |https://www.linkedin.com/in/abhishekkumaryadav26/|Shreya Yadav |https://www.linkedin.com/in/shreya-yadav-77b91324b|
|METAVERSE|https://github.com/apu52/METAVERSE|Arpan Chowdhury|Diya Bodiwala |www.linkedin.com/in/diya-bodiwala-234769290|Priyanka mewara |https://www.linkedin.com/in/Priyanka 50b995218/|
|Ethpreneur|**https://github.com/SpandanM110/Ethpreneur**|Ethpreneur|Parth Madhvani|https://www.linkedin.com/in/parthmadhvani2/|RUBADHARSINI S|https://www.linkedin.com/in/rubadharsini-saravanan-8a87aa24a|
|Blockchain based voting system|https://github.com/ishita-43/Blockchain-Based-Voting-System|Ishita|Om Prakash Chai|https://www.linkedin.com/in/omprakashchai/|R.D.Tarun|https://www.linkedin.com/in/rd-tarun/|
|Gluon|https://github.com/Makul0/gluon|Mukul | Dagar|SHRUTI PANDEY|https://www.linkedin.com/in/shruti-pandey-11a145222/|Rishitha Nandanapu|https://www.linkedin.com/in/rishitha-nandanapu-a61015301/|
|BitOptimus|https://github.com/siv2r/bit-optimus|Sivaram|Tushar|https://linkedin.com/in/tushar1504|Raghav Babbar|https://www.linkedin.com/in/babbar-raghav|
|CRAIBOT|https://github.com/Vansh-Choudhary/craibot-ai|Vansh Choudhary|Rohit Roy|https://www.linkedin.com/in/roy-rohit/|Ankit Mourya|https://www.linkedin.com/in/ankit-mourya-821aa223b/|
|Ether Wheels|https://github.com/StephCurry07/Ether-Wheels|Apoorv Gupta|Shivam Kumar Sinha|https://www.linkedin.com/in/sksinha2410|Kalyanasundaram K|https://www.linkedin.com/in/kalyan27/|
|wiki-3-dia|https://github.com/jaibhedia/wiki-3-dia|Shantanu Swami|HEMANTH KUMAR|https://www.linkedin.com/in/hemu21/|TANISHA GARG|https://www.linkedin.com/in/tanisha-garg-a97085299/|
|NexTrade|https://github.com/Akshat111111/NexTrade|Akshat Sharma|Pamidi Lakshmi Pavananjali|https://www.linkedin.com/in/lakshmi-pavananjali-pamidi-766760296|Jyoti kumari|https://www.linkedin.com/in/jyoti-kumari-b8bb67227|
|Fest RegCollegeistration Website|https://github.com/Ramsey99/fest-registration|Anuradha Adhikari|Sambit Mondal|https://www.linkedin.com/in/sambitm02|Sarthak Gupta |https://www.linkedin.com/in/sarthak-stranger|
|Sibananda Swain|https://github.com/sibananda0718/Blockchain-Based-Security-Management-System-|Sibananda Swain|Virengiri Goswami|https://www.linkedin.com/in/virengiri-goswami/|Aishwarya kathroju|www.linkedin.com/in/aishwarya-kathroju-a7448b265|
|Resources & Tools|https://github.com/Web3-0-World/Resources-Tools|Web3Sign|Jigar Prajapati|https://www.linkedin.com/in/jigar-prajapati-246102249/|Mastan Sayyad|https://www.linkedin.com/in/mastan-sayyad-126904223/|
|DecenTrade|https://github.com/4darsh-Dev/DecenTrade|Adarsh Maurya|Pratigya Panwar|https://www.linkedin.com/in/pratigya-panwar-37768b199|Radhika Malpani|https://www.linkedin.com/in/radhikamalpani1702/|
|blockchain|https://github.com/OpenTekHub/blockchain|Harish|Tanisha Bansal|https://www.linkedin.com/in/tanishabansal110902/|Amit Verma|` `https://www.linkedin.com/in/amit-verma-883150109/|
|Brave|https://github.com/edgexhq/brave|Subhadeep Roy|Ayishik Das |https://www.linkedin.com/in/ayishik-das|Unnati Gupta|https://www.linkedin.com/in/unnati-gupta-b73a8a27b|
|SupplyChainManagement|https://github.com/kartikmehta18/SupplyChainManagement|Kartik mehta|Aditya D|https://www.linkedin.com/in/aditya-d-23453a179/|Somnath Shaw|https://www.linkedin.com/in/somnath0904|
|Doctor-s-Care|https://github.com/GautamAnjali569/Doctor-s-Care|Anjali Gautam|Shashank Tomar |https://www.linkedin.com/in/iamshashanktomar|Garima Singh |www.linkedin.com/in/garima-singh-279014264|
|StakeFi|https://github.com/Parth-Sharma-tonchi/StakeFi|Parth Sharma|Ayush Tripathi|https://www.linkedin.com/in/ayush-t02/|Darshan Vasani|https://www.linkedin.com/in/dpvasani56/|
|Web3 Wallet|https://github.com/VedantAnand17/Web3-Wallet|Vedant Anand|Sivaram|https://www.linkedin.com/in/siv2r|HARSH SAINI|https://www.linkedin.com/in/harsh-saini-734ab6264/|
|CryptoX|https://github.com/Durgesh4993/CryptoX|Durgesh|Sanket Kalekar|https://www.linkedin.com/in/sanketkalekar/|Mohd Zaid Khan|https://www.linkedin.com/in/zeddkhan03|
|Decentralised Voting Dapp ( ICP mainnet )|https://github.com/ELiTE0005/Decentralized-Voting-System|RAJBIR BISWAS|Ritabrata Das|https://www.linkedin.com/in/ritabrata-das-6b0a40202|Het Modi|https://www.linkedin.com/in/het-modi-603728286|
|Gamesphere|https://github.com/ChromeGaming/GameSphere|Chrome Gaming|Akanksha Thaware|https://www.linkedin.com/in/akanksha-thaware|Aadil Latif|https://www.linkedin.com/in/iaadillatif|
|Cryptowallet|https://github.com/AbhiDiva96/CryptoPurse|Abhishek Kumar|Yash Daga |https://www.linkedin.com/in/yashdaga17/|Kartik mehta|https://www.linkedin.com/in/kartik-mehta-6729b0255/|
|SaveXross|https://github.com/SendouD/saveXross|JANANATHAN|Agamjot Singh |https://www.linkedin.com/in/agamjot-singh/|Trisha Sahu|https://www.linkedin.com/in/trisha-sahu-171623193/|
|cryptobot|https://github.com/OpenTekHub/cryptobot|Harish|AGRIMA SINGH|www.linkedin.com/in/agrima-singh-ba0067267|Rathod Shanker|https://www.linkedin.com/in/rathod-shanker-4a609421b|
|SNOWFLAKE-TEST|https://github.com/Durgesh4993/SNOWFLAKE-TEST|Durgesh|KANHAIYA KUMAR|https://github.com/iamkanhaiyakumar|Raza Ali|https://www.linkedin.com/in/muhammad-raza-ali-748425256|
|VaultChain|https://github.com/adarsh-singh01/VaultChain|Adarsh Singh|mansi ruhil|https://www.linkedin.com/in/mansi-ruhil-7a00a0228/|Anshita Soni |https://www.linkedin.com/in/anshita-soni-992270294|
|Community Page|https://github.com/Web3-0-World/Community-Page|Web3Sign|Bhuwan Saretia|https://www.linkedin.com/in/bhuwan-saretia/|Nupur Agarwal |https://www.linkedin.com/in/nupur-agarwal-43303725a|
|CryptoPurse|https://github.com/AbhiDiva96/CryptoPurse|Abhishek Kumar|Soujanya C |https://www.linkedin.com/in/soujanya-c-3a63a223a/|Jyotiraditya Upadhyay|https://www.linkedin.com/in/jyotiraditya-upadhyay-36a864236|
|saveXross|https://github.com/SendouD/saveXross|Jananathan|Kartikey Pandey|https://www.linkedin.com/in/kartikey-pandey-368497282/|Asmita Mishra|https://www.linkedin.com/in/asmitamishra1/|
|livetracker|https://github.com/mansiruhil13/Bobble-AI|Mansi Ruhil|Akshit Garg|https://www.linkedin.com/in/akshit-garg24/|Farha Kousar|https://www.linkedin.com/in/farhakousar16|
|Community-Page|https://github.com/ChromeGaming/Community-Page|Chrome Gaming|Kartik gurjar|https://www.linkedin.com/in/kartik-gurjar-37b26a309/|Suhasini singh|https://www.linkedin.com/in/suhasini-singh-bb49b0212|
|Online Quiz Application|https://github.com/Web3-0-World/Online-Qwiz-Application|Web3Sign|Md Azfar Alam|https://linkedin.com/in/md-azfar-alam/|Sanket Zambare|https://www.linkedin.com/in/sanketzambare/|
|College Fest Registration Website|https://github.com/Ramsey99/fest-registrationn|Anuradha Adhikari|Neal Gandhi|https://www.linkedin.com/in/neal-gandhi-a5b29526a|Sanikommu Devi|www.linkedin.com/in/sanikommu-devi-612a79251|
|Pay3|https://github.com/Anandprafull/Pay3|Vansh chauhan|Pranjali Bhardwaj|https://www.linkedin.com/in/pranjali-bhardwaj-a33ba2294/|K Bhanu Samyuktha |https://www.linkedin.com/in/sanyut-hem-02111331a|
|Cab Rentals|https://github.com/07sumit1002/CabRental|Sumit Kumar|Diya Agrawal |https://www.linkedin.com/in/diya-agrawal-nitp|Adarsh Chauhan|www.linkedin.com/in/adarsh-chauhan-b87609225|
|PyVerse|https://github.com/UTSAVS26/PyVerse|Mansi Ruhil|Abhishek Agarwal|https://www.linkedin.com/in/abhishek-agarwal-08695b306/|Akshit Batra|https://www.linkedin.com/in/akshit-batra|
|Builder Buddy|https://github.com/shikhar229169/Builder-Buddy|Shikhar Agarwal|Sudha|- |Mohammed Ufraan|www.linkedin.com/in/ufraaan|

**Comment by ishita-43** [comment]
Many links are broken in the above list example 
For blockchain based voting system link should be :- https://github.com/ishita-43/Blockchain-Based-Voting-System
And name should be Ishita instead of vishita

**Reply by Durgesh4993** [reply]
Now You can check

**Comment by UTSAVS26** [comment]
Hi @Durgesh4993 , there is an issue with the name of Project Admin for the Project **PyVerse**, i am the admin of that project *Utsav Singhal* but it is listed under the name *Mansi Ruhil*

**Comment by Meetjain1** [comment]
Yes @Durgesh4993 many links are broken it would be better if you add github username of admins by tagging them so that they can fix it may be they changed the repo name after applying in WoB . Thats why previous repo link is not working .

**Reply by Durgesh4993** [reply]
Now you check

**Reply by Meetjain1** [reply]
Yes now , its seems to be fine .

**Comment by UTSAVS26** [comment]
> Hi @Durgesh4993 , there is an issue with the name of Project Admin for the Project **PyVerse**, i am the admin of that project *Utsav Singhal* but it is listed under the name *Mansi Ruhil*

@Durgesh4993

**Comment by SwatiMishra-213** [comment]
I couldn't find my name in the list.
